tests/pytests/integration/modules/test_virt.py

```diff
@@ -10,6 +10,10 @@
 
 docker = pytest.importorskip("docker")
 
+pytestmark = pytest.mark.skipif(
+    True, reason="Skipping tests in 3002.x and 3003.x branches."
+)
+
```

Comment:
Doesn't this skip the test moving forward? If we only want to skip in those branches, we should remove this.